### ISSUE TITLE
Question: Generic che REST API client for browser

### ISSUE THREAD

**Issue by PavelSosin** [eclipse/che-workspace-client#10]
Does generic Che REST API client library for consumption in browser exist? or only
https://github.com/eclipse/che-workspace-client https://github.com/eclipse/che-workspace-client/blob/master/src/rest/remote-api.ts

**Comment by sunix** [comment]
yes this is what is the most generic, As far as i remember we also have one for GWT.
Then we have high level API for Theia IDE
https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-plugin
